src/index.js

```diff
@@ -84,6 +85,7 @@ export {
   Button,
   CircleButton,
   Radio,
+  RadioGroup,
```

Comment:
`RadioGroup` wasn't exported before?

Reply:
Really? Huh, I guess we don't have it anywhere in `customers`